org.adempiere.ui.zk/WEB-INF/src/web/theme/default/css/fragment/gadget.css.dsp

```diff
@@ -36,15 +36,15 @@
 .z-panel-head .z-panel-header {
 	padding: 0 0 2px 0;
 	color: #262626; 
-	background: #F4F4F4;
+	background: white; //changed from #F4F4F4 to white
 	font-weight: 300;
 	font-size: 13px;
 }	
 
 .z-caption .z-caption-content {
 	padding: 1px 1px 1px 6px;
 	color: #262626; 
-	font-weight: 300;
+	font-weight: 500; //changed from 300
```

Comment:
please remove this sort of comments that repeat what's already capture in git history.